internal/datastore/mysql/query_builder.go

```diff
@@ -65,7 +65,7 @@ func NewQueryBuilder(driver *migrations.MySQLDriver) *QueryBuilder {
 }
 
 func listCaveats(tableCaveat string) sq.SelectBuilder {
-	return sb.Select(colCaveatDefinition).From(tableCaveat).OrderBy(colName)
+	return sb.Select(colCaveatDefinition, colCreatedTxn).From(tableCaveat).OrderBy(colName)
```

Comment:
do we also need a new index here?

Reply:
I don't believe so, no

Reply:
potential concern is that this won't be a covering index anymore if it's not part of an existing index

Reply:
Possible, but I think that's already likely since the devision itself is not in the index